core/src/main/java/apoc/export/csv/CsvLoaderConfig.java

```diff
@@ -102,7 +110,7 @@ public static CsvLoaderConfig from(Map<String, Object> config) {
         if (config.get(SKIP_LINES) != null) builder.skipLines((int) config.get(SKIP_LINES));
         if (config.get(BATCH_SIZE) != null) builder.batchSize((int) config.get(BATCH_SIZE));
         if (config.get(IGNORE_DUPLICATE_NODES) != null) builder.ignoreDuplicateNodes((boolean) config.get(IGNORE_DUPLICATE_NODES));
-
+        if (config.get(IGNORE_EMPTY_STRING) != null) builder.ignoreEmptyString((boolean) config.get(IGNORE_EMPTY_STRING));
```

Comment:
nit: s/IGNORE_EMPTY_STRING/IGNORE_BLANK_STRING/ ?

Reply:
Sorry, I didn't get it 😅.
You mean another config `ignoreBlankString()`?

Reply:
My bad, I meant, rename `IGNORE_EMPTY_STRING` to `IGNORE_BLANK_STRING` (but you're right about renaming `builder#ignoreEmptyString` too!)